test/TestProjects/Accessibility-LowLevel-TokenAuth/Generated/AccessibilityClient.cs

```diff
@@ -32,15 +32,22 @@ protected AccessibilityClient()
         {
         }
 
+        /// <summary> Initializes a new instance of AccessibilityClient. </summary>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="credential"/> is null. </exception>
+        public AccessibilityClient(TokenCredential credential) : this(credential, new Uri("http://localhost:3000"), new AccessibilityClientOptions())
+        {
+        }
+
         /// <summary> Initializes a new instance of AccessibilityClient. </summary>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="endpoint"> server parameter. </param>
         /// <param name="options"> The options for configuring the client. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="credential"/> is null. </exception>
-        public AccessibilityClient(TokenCredential credential, Uri endpoint = null, AccessibilityClientOptions options = null)
+        /// <exception cref="ArgumentNullException"> <paramref name="credential"/> or <paramref name="endpoint"/> is null. </exception>
+        public AccessibilityClient(TokenCredential credential, Uri endpoint, AccessibilityClientOptions options = null)
```

Comment:
Why `options` is default in here? I see in some projects it is required.

Reply:
I followed this https://github.com/Azure/autorest.csharp/issues/2055#issuecomment-1076868591, but it seems still under discussion.